Controllers/HomeController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            ViewData["url"] = "https://bagcupid.com/";
+            ViewData["type"] = "website";
+            ViewData["title"] = "Bag Cupid";
+            ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+
+            return View();
+        }
+
+        [Route("app", Name = "app", Order = 0)]
+        public IActionResult app()
+        {
+            ViewData["url"] = "https://bagcupid.com/";
+            ViewData["type"] = "website";
+            ViewData["title"] = "Bag Cupid";
+            ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+
+            return View("Index");
+        }
+
+        [Route("app/product/{productId}", Name = "product", Order = 0)]
+        public IActionResult product(int productId)
+        {
+            ViewData["url"] = "Muhammad Nabeel";
+            ViewData["type"] = "Managing Director";
+            ViewData["title"] = "Principal Software Engineer / Technical Lead";
+            ViewData["description"] = "Relliks Systems";
+
+            return View("Index");
+        }
+    }

```

Comment:
It seems that this file has its indentation fixed, but the line ending fix didn't take effect.  :/  I have made a change to the upstream repository (`Zoltu/bags-client`) that should make git do the right thing in the future, but in order to get that you will need to revert the change you made to `.gitignore` and then do a rebase on `origin/master`.  I can work with you on how to do that, or there are docs online for how to do that.  However, that won't resolve the line ending problem right now, only in the future.

The easiest solution is to install this extension: https://visualstudiogallery.msdn.microsoft.com/b2bbadd2-c337-43d7-9343-752ebbdd900f and right click in Visual Studio on the files you added/changed and unify the line endings to `Linux (LF)`.